### DISCUSSION TITLE
Bounding Box Feature

### DISCUSSION THREAD

**Discussion by anais2390** [Project-MONAI/MONAI#3354] [Q&A]
Hi MONAI Team, 

I was wondering if you're considering adding a bounding box training feature to the MONAI library for 'object' detection. 
Such as Mask RCNN network and data augmentation compatbile with bounding box labelization. 
It would be helpfull in lesion detection or occlusion detection in medical images for exemple
Thanks !

**Comment by Nic-Ma** [comment]
Hi @anais2390 ,

Thanks for your feedback.
Detection task is a big feature request, we are already discussing it and plan to support in later versions.

Thanks.

**Reply by anais2390** [reply]
Thanks @Nic-Ma , this is a good news ! Do you have any idea of when this feature would be available ?

**Reply by wyli** [reply]
@Can-Zhao is leading this feature development, we aim at a preliminary release in first quarter 2022.

**Reply by rgaiacs** [reply]
@Can-Zhao any update regarding object detection? I'm starting a new project and deciding if I should use MONAI or something else.

**Reply by Can-Zhao** [reply]
Detection feature will be added in June.
